client/state/plugins/wporg/reducer.js

```diff
@@ -45,7 +71,58 @@ export function items( state = {}, action ) {
 	}
 }
 
+export function lists( state = {}, action ) {
+	const { category, data, searchTerm, type } = action;
+	switch ( type ) {
+		case PLUGINS_WPORG_LIST_RECEIVE:
+			if ( data ) {
```

Comment:
What about using an early-return `if ( ! data ) { return state; }` in this case to avoid one level of nesting?

Reply:
Good call. Done in 9fad4151ccd482e33e810f22e12fd27430a27eb7